alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java

```diff
@@ -23,10 +23,7 @@
 import java.io.ByteArrayOutputStream;
 import java.util.*;
 
-import org.apache.zeppelin.interpreter.Interpreter;
-import org.apache.zeppelin.interpreter.InterpreterContext;
-import org.apache.zeppelin.interpreter.InterpreterPropertyBuilder;
-import org.apache.zeppelin.interpreter.InterpreterResult;
+import org.apache.zeppelin.interpreter.*;
```

Comment:
Would you like to keep import block unchanged here (and the in the other interpreters)?